src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
+  const props = sourceString.split(';').map((prop) => prop.trim());
+
+  for (const prop of props) {
+    const [key, value] = prop.split(':');
```

Comment:
The current implementation splits each property by the first colon found. If a property contains multiple colons, this will not work as expected. Consider using a more robust method to handle such cases, or document that the input format should not contain multiple colons in a single property.